src/main/scala/beam/agentsim/agents/vehicles/BeamVehicle.scala

```diff
@@ -554,6 +555,9 @@ object BeamVehicle {
   def isSharedTeleportationVehicle(vehicleId: Id[BeamVehicle]): Boolean = {
     vehicleId.toString.startsWith(idPrefixSharedTeleportationVehicle)
   }
+  def isSharedVehicle(vehicleId: Id[BeamVehicle]): Boolean = {
+    vehicleId.toString.startsWith(idPrefixSharedVehicle)
+  }
```

Comment:
If we use `BeamVehicle.isSharedVehicle` as suggested above we won't need this any more

Reply:
true, deleted